src/components/GroceryItem.svelte

```diff
@@ -0,0 +1,146 @@
+<script>
+  export let item;
+  export let removeItem;
+  export let updateItem;
+  export let editItem;
+  export let itemId;
+  export let isEditing = false;
+
+  export let inputEl;
+  let trashIconImage = "./images/trash.svg";
+  let editIconImage = "./images/edit-icon.svg";
+</script>
+
+<li class="item">
+  {#if isEditing}
+    <input class="input-update" type="text" bind:value={inputEl} />
+    <button
+      class="update-btn"
+      on:click={() => {
+        updateItem(itemId, inputEl);
+      }}>Update</button
+    >
+  {:else}
+    <span class="name">{item.name}</span>
+    <button
+      class="edit-item-btn btn-white btn-animate"
+      on:click={() => {
+        editItem();
+      }}><img src={editIconImage} alt="{editIconImage} image" /></button
+    >
+  {/if}
+
+  <button class="remove-item-btn" on:click={() => removeItem(itemId)}
+    ><img src={trashIconImage} alt="{trashIconImage} image" /></button
```

Comment:
People who use screen readers to navigate a webpage won't know the purpose of this button because it doesn't has text, only icons (in this case an image). 

This image is a [functional image](https://www.w3.org/WAI/tutorials/images/functional/). That means it's used as an action, in this case, to delete the item.

So, the `alt` should reflect that. We can do `alt="Delete product".`

That way, people using screen readers, when they reach this button it will announce `"button, delete product image"`. 

Please review the buttons that are only an image and be mindful about their accessible name :)

Reply:
@sandrina-p Thanks for the resource, that is good to know :).